phase2-cli/src/verify.rs

```diff
@@ -54,9 +56,14 @@ pub fn verify(
     info!("`response` file contains decompressed points and has a hash:");
     print_hash(&response_hash);
 
+    let after_compressed = if verifying_full_contribution {
```

Comment:
isn't this too specific? why not have this as a parameter so the caller can decide the compression?

Reply:
It gives the caller a guarantee that the compression will line up correctly as long as you pass in correctly what you want to do with a function call. Otherwise you'd need to go through your code to make sure every compression parameter lines up with every other API call, which could be tedious and error-prone

Reply:
Doesn't it impose a requirement that the "combine" code also outputs with specific compression? While it is not concretely limited?

Reply:
Good point. I refactored the compression constants so there are only 3 now, located in `phase2-cli/lib.rs. This guarantees i.e. combine always outputs the compression expected by verify for full contributions